packages/main/src/Link.js

```diff
@@ -115,6 +115,16 @@ const metadata = {
 			defaultValue: "",
 		},
 
+		/**
+		 * @type {String}
+		 * @defaultvalue ""
+		 * @private
+		 * @since 1.0.3
```

Comment:
You would need to update the since a bit. This is a feature, can't be included in patch 1.0.3, it will be included in 1.1.0 version

Reply:
Done